src/App.tsx

```diff
@@ -9,24 +9,22 @@ interface Todo {
 function App() {
   const [items, setItems] = useState<Todo[]>([]);
 
+  const todos = items.reduce((acc: any[], obj) => {
+    const values = Object.values(obj).slice(1, 3);
+    return [...acc, values];
+  }, []);
+
   useEffect(() => {
     fetch('https://jsonplaceholder.typicode.com/todos')
       .then((response) => response.json())
       .then(setItems);
```

Comment:
"todos" array should be manipulated at ".then" stage, otherwise the same code will be executed many times.

Reply:
Done